tests/assembly/issue-83585-small-pod-struct-equality.rs

```diff
@@ -0,0 +1,29 @@
+//@ assembly-output: emit-asm
+//@ compile-flags:-Copt-level=3
+//@ only-x86_64
+
+#![crate_type = "lib"]
+
+#[no_mangle]
+type T = u8;
+type T1 = (T, T, T, T, T, T, T, T);
+type T2 = [T; 8];
+
+#[no_mangle]
```

Comment:
I think it it more idiomatic to have `#[no_mangle]` directly before the function, without comments interleaved.